### PR TITLE
Add support of ignorecase for MySQL by using `lower()`

### PR DESCRIPTION
**Pull request by LoicMahieu**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by ssh24** [comment]
@LoicMahieu Thank you for the contribution! Could you please rebase your branch or give me the access to your fork so that I could rebase for you? More info on how to give access of your fork: https://github.com/blog/2247-improving-collaboration-with-forks

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by jannyHou** [comment]
@LoicMahieu Thank you for contribution, any update on code rebase?

File: lib/mysql.js [hunk 1]
```diff
@@ -462,17 +462,20 @@ MySQL.prototype.ping = function(cb) {
 MySQL.prototype.buildExpression = function(columnName, operator, operatorValue,
     propertyDefinition) {
   if (operator === 'regexp') {
-    if (operatorValue.ignoreCase)
-      console.warn('MySQL regex syntax does not respect the `i` flag');
+    var clause = columnName + ' REGEXP ?'
```
Comment (b-admike) [inline lib/mysql.js]: `;` at the end here

**Comment by ssh24** [comment]
Closing this PR as it does not achieve anything. We currently do not have support for `like` and `nlike` operations. Will add a ticket on the backlog for that feature.

**Pull request closed without merging**